lib/core/src/model.rs

```diff
@@ -879,6 +888,7 @@ pub(crate) struct SendSwap {
     /// Persisted as soon as a refund tx is broadcast
     pub(crate) refund_tx_id: Option<String>,
     pub(crate) created_at: u32,
+    pub(crate) timeout_block_height: u64,
```

Comment:
We have the create_response_json that already contains the timeout_block_height. I am wondering if we need to persist the timeout_block_height separately in addition or we can just extract the value from the create_response when needed.

Reply:
It's also been done for other fields, and I'm not against it since it saves us from parsing the create response every time we display a payment

Reply:
Yes, I missed the point it was our way also with other fields. I am fine with it then.